File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string ans = "";
        int i = 0;
        while (i < word1.length() || i < word2.length()) {
            if (i < word1.length()) {
                ans += word1[i];
            }
            if (i < word2.length()) {
                ans += word2[i];
            }
            i++;
        }
        return ans;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        min_len = min(len(word1),len(word2))
        merged = []
        for i in range(0,min_len):
            merged.append(word1[i])
            merged.append(word2[i])
        merged.extend(word1[min_len:])
        merged.extend(word2[min_len:])

        return ''.join(merged)